Makefile.calico-node

```diff
@@ -340,7 +345,8 @@ node-ut:
 
 PHONY: node-test-containerized
 ## Run the tests in a container. Useful for CI, Mac dev.
-node-test-containerized:
+node-test-containerized: vendor run-etcd-host
 	docker run --rm -v $(CURDIR):/go/src/$(PACKAGE_NAME):rw \
-    -e LOCAL_USER_ID=$(LOCAL_USER_ID) \
-    $(CALICO_BUILD) sh -c 'cd /go/src/$(PACKAGE_NAME) && make node-ut'
+	-e LOCAL_USER_ID=$(LOCAL_USER_ID) \
+	--net=host \
```

Comment:
It's probably slightly better if we can avoid usinig `--net=host` for test containers. It helps to avoid polluting the host namespace if/when things fail and don't get cleaned up properly.